web/app/themes/cfs/templates/article-person.php

```diff
@@ -1,18 +1,29 @@
 <?php
+$person_post->meta = get_post_meta($person_post->ID);
 $person_title = get_post_meta($person_post->ID, '_cmb2_person_title', true);
+$person_show_page = get_post_meta($person_post->ID, '_cmb2_show_link', true);
+$person_image = \Firebelly\Media\get_header_bg($person_post, ['size' => 'medium']);
+$person_desc = strip_shortcodes( $person_post->ID );
 $person_desc = apply_filters('the_content', $person_post->post_content);
-$person_image = \Firebelly\Media\get_header_bg($person_post, ['size' => 'medium'])
+$person_desc = str_replace(']]>', ']]&gt;', $person_desc);
+$excerpt_length = apply_filters( 'excerpt_length', 25 );
+$excerpt_more = apply_filters( 'excerpt_more', ' ' . '[&hellip;]' );
```

Comment:
```suggestion
$excerpt_more = apply_filters( 'excerpt_more', $person_post );
```
Passing the person post will ensure we get the proper permalink back in the continue link